README.md

```diff
@@ -54,3 +54,12 @@ userFilter -- list of values which must be present in given field in filtered da
 \*\* -- immune to filters (can see everything)
 Public -- users with no userFilter are assigned this filter
 An item with no filter value is returned in all cases, and is thus also public.
+
+## Local Development Environment
+In order to quickly setup a development environment, make use of the `setup_script.sh` script. This will setup the project, initialize and seed the database configurations, import routes and initialize environment config files and generate the keys required.
+
+In order to run the setup script, 
+- Make the file executable via `chmod +x setup_script.sh`
```

Comment:
you _should_ be able to tell git that a script is executable (for ex, see https://stackoverflow.com/questions/21691202/how-to-create-file-execute-mode-permissions-in-git-on-windows)
If that doesn't end up working, then we can just leave this.

Reply:
I've followed the reference provided and did the same for the setup script, thanks !